csclient/params/params.go

```diff
@@ -449,3 +449,24 @@ type SetAuthCookie struct {
 	// Macaroons holds a slice of macaroons.
 	Macaroons macaroon.Slice
 }
+
+// UploadIdResponse holds the upload id response from a POST request to /upload.
+type UploadIdResponse struct {
+	UploadId string
+}
+
+// Parts holds the list of part for an upload to be completed.
```

Comment:
// Parts holds a list of all the parts that are required by a multipart
// upload, as required by a PUT request to /upload/$upload-id.

?